centos-ci/README.md

```diff
@@ -0,0 +1,37 @@
+# LeApp CI
+
+LeApp CI is hosted on the [CentOS CI infrastructure](http://ci.centos.org/) using Jenkins and since
+our tests are virtualization heavy we have a dedicated slave node (slave03) for our tests.
+
+## Architecture
+
+When Jenkins is triggered via a webhook it uses a Duffy key located in `~/duffy.key` to get a node from the CICO
+cluster on which the test will run. Duffy is CentOS CI middleware for resource management inside the cluster.
+The lifetime (read, debuggability) of a failed test run is 12 hours.
+
+## Configuration
+
+All configuration regarding the CI can be found in the `centos-ci` directory where this readme is located.
+Secrets and SSH keys can be found in the the housekeeping repository.
+The directory structure of `centos-ci` is as follows:
+
+| File/Directory | Purpose |
+|----------------|---------|
+| `ansible` | Playbook and roles setting up the test environment |
+| `jobs/integration_test.yml` | Jenkins Job Builder definition for our Jenkins job, more explanation below |
+| `patches` | Any patches that we currently carry to make sure that test environment works |
+| `bootstrap.sh` | Entry point of our CI, this is the first executed in a new test run |
+
+## Jenkins Job
+
+Jenkins Job Builder file is a declarative way (YAML) of describing Jenkins Job, this file also describes who
+can start test runs and what trigger phrase to use for PRs not by people from the admin list.
+
+### Adding yourself to the list of people who can trigger a test run
+
+Add your GitHub handle to the `admin-list` list under `jobs.triggers.github-pull-request` and open a PR with the change.
+
+### The trigger phrase
+
+We're currently using `*Jenkins Test*` with asterisks being wildcards, when any user in the `admin-list` comments on a PR
```

Comment:
Could we make the command phrase a bit more obviously a command or directive, rather than an ordinary noun phrase? I'm thinking something like: "Jenkins: ok to test"